R/ScenarioConfiguration.R

```diff
@@ -0,0 +1,181 @@
+#' @title ScenarioConfiguration
+#' @docType class
+#' @description An object storing configuration of a specific scenario
+#' @format NULL
+ScenarioConfiguration <- R6::R6Class(
+  "ScenarioConfiguration",
+  inherit = ospsuite.utils::Printable,
+  cloneable = TRUE,
+  active = list(
+    #' @field setTestParameters Boolean representing whether parameters defined in \code{TestParameters} are to be applied
+    #' to the simulation
+    setTestParameters = function(value) {
+      if (missing(value)) {
+        private$.setTestParameters
+      } else {
+        ospsuite.utils::validateIsLogical(value)
```

Comment:
We had decided against namespacing `ospsuite.utils::` because they are so frequently used, and there is unlikely to be a namespace collision with any other package we are relying on.